waveguide_mode_converter/mode_converter_meep_opt.py

```diff
@@ -63,9 +63,9 @@
 Si = mp.Medium(index=nSi)
 
 design_region_size = mp.Vector3(dx, dy, 0)
-design_region_resolution = int(2 * resolution)
-Nx = int(design_region_size.x * design_region_resolution)
-Ny = int(design_region_size.y * design_region_resolution)
+design_region_resolution = 99.375
+Nx = int(round(design_region_size.x * design_region_resolution)) + 1
+Ny = int(round(design_region_size.y * design_region_resolution)) + 1
```

Comment:
```suggestion
design_region_resolution = int(2 * resolution)
Nx = int(design_region_size.x * design_region_resolution)
Ny = int(design_region_size.y * design_region_resolution)
design_region_resolution = (Nx - 1) / design_region_size.x # correct for +1 factor in recent Meep
```

Reply:
You could also use
```py
design_region_resolution = (Nx - 1 + 1e-8) / design_region_size.x 
```
to compensate for possible roundoff errors, to ensure that `int(design_region_size.x * design_region_resolution) + 1 == Nx`

Reply:
Since the filter code calls `round`, this should not be needed.